cloud: update Backup Setting page (#20950)
pingcap/docs: tidb-cloud/backup-and-restore.md

1. Navigate to the **Backup** page of a TiDB Cloud Dedicated cluster.

2. Click **Backup Setting**.

3. Toggle the **Auto Backup** switch to **On**.

4. Configure the backup schedule as follows:

>     - In **Backup Cycle**, select either the **Daily Backup** or **Weekly Backup** tab. If you select **Weekly Backup**, you need to specify the days of the week for the backup.

Reviewer: ```suggestion
    - In **Backup Cycle**, click either the **Daily Backup** or **Weekly Backup** tab. For **Weekly Backup**, you need to specify the days of the week for the backup.
```

1. Navigate to the **Backup** tab of a cluster.

> 2. Click **...** > **Manual Backup**. The setting window displays.

Reviewer: ```suggestion
2. In the upper-right corner, click **...** > **Manual Backup**. 
```

> 3. Enter a **Name**.

Reviewer: ```suggestion
3. In the displayed dialog, enter a **Name**.
```

> 1. Navigate to the **Backup** tab of a cluster.

Reviewer: ```suggestion
1. Navigate to the **Backup** page of a cluster.
```

2. Locate the corresponding backup file you want to delete, and click **...** > **Delete** in the **Action** column.

#### Delete a running backup job

To delete a running backup job, it is similar as [**Delete backup files**](#delete-backup-files).

> 1. Navigate to the **Backup** tab of a cluster.

Reviewer: ```suggestion
1. Navigate to the **Backup** page of a cluster.
```
